packages/renderer/src/lib/ui/ListItemButtonIcon.svelte

```diff
@@ -6,13 +6,12 @@ export let title: string;
 export let icon: IconDefinition;
 export let hidden: boolean = false;
 export let onClick: () => void = () => {};
-export let backgroundColor: string = 'bg-zinc-800';
 </script>
 
 <button
   title="{title}"
   on:click="{onClick}"
-  class="mx-1 text-gray-300 {backgroundColor}  hover:text-violet-600 font-medium rounded-lg text-sm inline-flex items-center px-3 py-2 text-center"
```

Comment:
no need to use backgroundColor anymore since we don't really need it UI-wise. feel free to give your input on this @benoitf

Reply:
It's used ui-wise in the details of a container
So changing the actions changed as well the rendering in the details
and here we don't see anymore the background that was a parameter before
![image](https://user-images.githubusercontent.com/436777/200768557-6e025ca3-fe53-4641-8efc-d0432ea88ce3.png)

I'm not sure we need to have kebab menu in the details as we've much more space around (we could depending on the number of actions to be displayed)